docs/sources/articles/https.md

```diff
@@ -101,8 +101,8 @@ config file:
 
 Now sign the public key:
 
-    $ openssl x509 -req -days 365 -in client.csr -CA ca.pem -CAkey ca-key.pem \
-      -CAcreateserial -out cert.pem -extfile extfile.cnf
+    $ openssl x509 -req -days 365 -sha256 -in client.csr -CA ca.pem -CAkey ca-key.pem \
+      -CAcreateserial -out client-cert.pem -extfile extfile.cnf
```

Comment:
Oh! Sorry, I overlooked this earlier, but you changed to `client-cert.pem` here, but the documentation below uses `cert.pem`. Could you change this back?